pyproject.toml

```diff
@@ -39,34 +45,30 @@ dependencies = [
 [project.optional-dependencies]
 train = [
     "accelerate>=0.30.1",
-    "numpy==1.26.4",
     "bitsandbytes",
     "configue>=5.0.0",
     "datasets>=2.19.1",
-    "mteb>=1.12.22",
```

Comment:
Pretty sure you need mteb here to run evaluation at the end of training

Reply:
Yup but I've moved it to the compulsory deps so it always gets installed